libs/application-generic/src/usecases/tier-restrictions-validator/tier-restrictions-validator.command.ts

```diff
@@ -0,0 +1,14 @@
+import { IsEnum, IsNumber, IsOptional } from 'class-validator';
+
+import { StepTypeEnum } from '@novu/shared';
+import { OrganizationLevelCommand } from '../../commands';
+
+export class TierRestrictionsValidatorCommand extends OrganizationLevelCommand {
```

Comment:
shouldn't usecase describe an action and a verb

Reply:
good point i will update it